Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -52,7 +52,8 @@
    "outputs": [],
    "source": [
     "# Specify the name of the user mounted directory\n",
-    "user_mounted_dir_name = \"user\""
+    "user_mounted_dir_name = \"user\"\n",
+    "airgap_registry=\"lr1-bd-harbor-registry.mip.storage.hpecorp.net/develop/\""
```

Comment:
We need to get it from env var

Reply:
Added the airgap_registry from env in the base_image path